pkg/controller/node/nodecontroller.go

```diff
@@ -215,7 +215,7 @@ func NewNodeController(
 	runTaintManager bool,
 	useTaintBasedEvictions bool) (*NodeController, error) {
 	eventBroadcaster := record.NewBroadcaster()
-	recorder := eventBroadcaster.NewRecorder(api.Scheme, clientv1.EventSource{Component: "controllermanager"})
+	recorder := eventBroadcaster.NewRecorder(api.Scheme, clientv1.EventSource{Component: "node-controller"})
```

Comment:
[here](https://github.com/knightXun/kubernetes/blob/b939199fcf05584624ae734b57122c5a17539751/pkg/controller/node/taint_controller.go#L155) and other possible places.